automation/build-artifacts.repos.el7

```diff
@@ -1,2 +1,3 @@
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs-modules_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
+http://jenkins.ovirt.org/job/ovirt-engine-yarn_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
```

Comment:
@eedri , @ifireball , The project needs to be built against defined ovirt-engine-nodejs-modules package version. How to achieve that, please?